Code/training_common_utils.py

```diff
@@ -4,15 +4,26 @@
 import pandas as pd
 import numpy as np
 import tensorflow as tf
-from CheXpertDataset import ChexpertDataset, UZerosTransform, UOnesTransform, ReplaceNaNTransform, GreyScale_to_RGB
+from CheXpertDataset import ChexpertDataset, \
+    UZerosTransform, UOnesTransform, ReplaceNaNTransform
+from ImageTransforms import PILToNumpy, FixRatioResize, EqualizeHistogram, Gray2RGB, GaussianBlur
 from torchvision import transforms
 from torch.utils.data import Subset, DataLoader
 
 image_preprocessing = transforms.Compose([
-    # GreyScale_to_RGB(),
-    transforms.Resize((600, 600)),
+    PILToNumpy(),
+    EqualizeHistogram(),
+    Gray2RGB(),
+    FixRatioResize(512, 128),
+    GaussianBlur(3),
     transforms.ToTensor(),
-    transforms.Normalize((0, 0, 0), (1.0, 1.0, 1.0))
+    transforms.Normalize((128, 128, 128), (64, 64, 64))
+])
+
+image_augmented_preprocessing = transforms.Compose([
+    transforms.RandomAffine(degrees=(-15, 15), translate=(0.05, 0.05),
+                            scale=(0.85, 1.05), fillcolor=128),
+    image_preprocessing
 ])
```

Comment:
Order and settings for both the augmented and normal data preprocessing steps came from the order in which they were applied in the Chexpert winners solution.

Reply:
Note that image sizes should be 512x512 moving forward.